### PR TITLE
feat(appeals-fo): add listed building topic

### PR DESCRIPTION
**Pull request by bryn-pins**
added service bus topic to handle updates for listed buildings
intended as a temporary solution until ODW has their own topic we can subscribe to

### CODE REVIEW

File: app/stacks/uk-west/appeals-service/terragrunt.hcl [hunk 1]
```diff
@@ -26,21 +26,6 @@ dependency "common_ukw" {
   }
 }
 
-dependency "back_office_ukw" {
```
Comment (bryn-pins) [inline app/stacks/uk-west/appeals-service/terragrunt.hcl:29]: this appears to be an old duplicate of the section below

File: app/components/appeals-app-services/function-app.tf [hunk 2]
```diff
@@ -81,3 +81,12 @@ resource "azurerm_servicebus_subscription" "appeals_bo_document_topic_subscripti
   topic_id           = var.service_bus_appeals_bo_document_topic_id
   max_delivery_count = 1
 }
+
+# ODW Listed Building topic subscription
+resource "azurerm_servicebus_subscription" "listed_building_topic_subscription" {
+  count = var.appeals_feature_back_office_subscriber_enabled ? 1 : 0
```
Comment (bryn-pins) [inline app/components/appeals-app-services/function-app.tf:87]: resuing appeals_feature_back_office_subscriber_enabled but could potentially add a separate feature toggle

File: app/stacks/uk-south/appeals-service/terragrunt.hcl [hunk 3]
```diff
@@ -77,6 +78,7 @@ inputs = {
   back_office_service_bus_namespace_name          = dependency.back_office_ukw.outputs.service_bus_namespace_name
   service_bus_appeals_bo_case_topic_id            = dependency.back_office_ukw.outputs.service_bus_appeals_bo_case_topic_id
   service_bus_appeals_bo_document_topic_id        = dependency.back_office_ukw.outputs.service_bus_appeals_bo_document_topic_id
+  service_bus_odw_listed_building_topic_id        = dependency.back_office_ukw.outputs.service_bus_odw_listed_building_topic_id
```
Comment (bryn-pins) [inline app/stacks/uk-south/appeals-service/terragrunt.hcl:81]: there appears to be a mix of ukw and uks references in here
thinking this should potentially be fixed in another PR

File: app/stacks/uk-west/appeals-service/terragrunt.hcl [hunk 4]
```diff
@@ -64,6 +50,7 @@ inputs = {
   back_office_service_bus_namespace_name    = dependency.back_office_ukw.outputs.service_bus_namespace_name
   service_bus_appeals_bo_case_topic_id      = dependency.back_office_ukw.outputs.service_bus_appeals_bo_case_topic_id
   service_bus_appeals_bo_document_topic_id  = dependency.back_office_ukw.outputs.service_bus_appeals_bo_document_topic_id
+  service_bus_odw_listed_building_topic_id  = dependency.back_office_ukw.outputs.service_bus_odw_listed_building_topic_id
```
Comment (chrishawldev) [inline app/stacks/uk-west/appeals-service/terragrunt.hcl:53]: Maybe just remove the odw from the name to avoid confusion

File: app/stacks/uk-west/appeals-service/variables.tf [hunk 5]
```diff
@@ -251,6 +251,12 @@ variable "service_bus_appeals_bo_document_topic_id" {
   description = "ID for the appeals case data topic"
   type        = string
 }
+
+variable "service_bus_odw_listed_building_topic_id" {
```
Comment (chrishawldev) [inline app/stacks/uk-west/appeals-service/variables.tf:255]: Maybe just remove the odw from the name to avoid confusion

File: app/stacks/uk-west/back-office/outputs.tf [hunk 6]
```diff
@@ -166,6 +166,11 @@ output "service_bus_nsip_project_update_topic_id" {
   value       = azurerm_servicebus_topic.nsip_project_update.id
 }
 
+output "service_bus_odw_listed_building_topic_id" {
```
Comment (chrishawldev) [inline app/stacks/uk-west/back-office/outputs.tf:169]: Maybe just remove the odw from the name to avoid confusion

File: app/stacks/uk-west/back-office/service-bus.tf [hunk 7]
```diff
@@ -149,3 +149,10 @@ resource "azurerm_servicebus_topic" "appeal_bo_document" {
   namespace_id        = azurerm_servicebus_namespace.back_office.id
   default_message_ttl = "P14D"
 }
+
+# ODW Producers
```
Comment (chrishawldev) [inline app/stacks/uk-west/back-office/service-bus.tf:153]: Maybe just remove the odw from the comment to avoid confusion

**Review by chrishawldev — COMMENTED** [review]
Just some feedback on naming conventions but otherwise looks good

**Review by chrishawldev — APPROVED** [review]
